uaclient/tests/test_cli_detach.py

```diff
@@ -94,3 +95,25 @@ def test_returns_zero(self, m_entitlements, m_getuid):
         ret = action_detach(mock.MagicMock(), mock.MagicMock())
 
         assert 0 == ret
+
+    @mock.patch("uaclient.cli.entitlements")
+    def test_informational_message_emitted(
+        self, m_entitlements, m_getuid, capsys
+    ):
+        m_getuid.return_value = 0
+        m_entitlements.ENTITLEMENT_CLASSES = [
```

Comment:
Would like to see a test as well, or parameterize the existing test to validate the plural services vs service: